routes/web.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('/', function () {
+    return view('welcome');
+});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Closing parenthesis of a multi-line function call must be on a line by itself](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)